src/components/Avatar/tests/Avatar-ssr.test.tsx

```diff
@@ -3,11 +3,13 @@ import React from 'react';
 import {mountWithAppProvider} from 'test-utilities/legacy';
 import {Avatar, Image} from 'components';
 
-jest.mock('../../../utilities/target', () => ({
-  get isServer() {
-    return true;
-  },
-}));
+jest.mock('../../../utilities/use-is-after-initial-mount', () => {
```

Comment:
It doesn't feel like this test adds much value anymore, mocking out individual hooks feels like a weird smell. Do we care that much about initial renderings?


Strictly speaking this isn't even testing SSR rendering - it's a bad proxy of it by mocking out a single hook. I wonder if ideally react-testing should provide some version of `mountOnServer` that renders a component using `react-dom/server` (like the hack described in https://github.com/testing-library/react-testing-library/issues/561), but that feels like a meaty problem for another day.


---

On a very petty note, we can save the explicit return:

```
jest.mock('../../../utilities/use-is-after-initial-mount', () => ({
  useIsAfterInitialMount: () => false,
}));
```

Reply:
To me it provides the same value as before. Earlier we were pretending to be on a server environment by mocking a constant now we simulate that by mocking a hook. Even though I agree earlier it was more explicit. I also agree that a `renderOnServer` function that doesn't mount would be pretty great for these cases. Do you want me to delete the test?

As for the `petty note`: I generally try to avoid to use single line arrow functions for two reasons. The `()` in `({})` easy to forget, which leads to the function not returning anything. And secondly, if you want to add a second line, it's quite tedious to make that happen and I don't see any advantage of using the single line notation in the first place. 
That being said, it's personal preference and I'd be open to change it.

Reply:
If you think it's still got some value let's keep it :)

Same goes for the petty note bits - you've got a solid rational, and it's personal pref vs idle musings.

Keep it all as it is and :shipit: :)